tpl/bootstrap.php

```diff
@@ -489,10 +489,17 @@
 				</div>
 <?php
 endif;
-?>
-				<div id="status" role="alert" class="alert alert-info<?php echo empty($STATUS) ? ' hidden' : '' ?>">
+?>					
+				<div id="status" role="alert" class="clearfix alert alert-<?php echo (bool)$ISDELETED ? 'success' : 'info'; echo empty($STATUS) ? ' hidden' : '' ?>">
```

Comment:
I am curious… What's `clearfix`?

Reply:
It is used to clear floats for the element ([ref](https://getbootstrap.com/docs/3.3/css/#helper-classes-clearfix)) so that the `pull-right` in the child button can work properly.

This is how it looks without `clearfix`
![alert-without-clearfix](https://github.com/user-attachments/assets/c0c334d5-7fbc-473c-8ed4-8357e909e0ae)

Reply:
Hint:
The `pull-right` class does no longer exist since Bootstrap 4 » https://stackoverflow.com/questions/39655110/what-happened-to-the-pull-left-and-pull-right-classes-in-bootstrap-4

And since the project moved to Bootstrap 5, this might need an overhaul.

Reply:
That's right. I'm only using the `pull-right` in the Bootstrap 3 template. The Bootstrap 5 doesn't have any of either `clearfix` or `pull-right`